sidebars.ts

```diff
@@ -915,6 +900,21 @@ module.exports = {
         'manage/security/set-max-web-session-timeout',
        ]
      },
+     {
+      type: 'category',
+      label: 'Views',
+      collapsible: true,
+      collapsed: true,
+      link: {type: 'doc', id: 'manage/views/index'},
+      items: [
+        'manage/views/add-scheduled-view',
+        'manage/views/pause-disable-scheduled-views',
+        'manage/views/run-search-against-scheduled-view',
+        'manage/views/scheduled-view-lag-time',
+        'manage/views/scheduled-views-best-practices',
+        'manage/views/view-list-scheduled-views',
```

Comment:
See this comment https://github.com/SumoLogic/sumologic-documentation/pull/4522/files#r1756162252